### DISCUSSION TITLE
Add option to bulk delete or replace duplicate codes prior to import

### DISCUSSION THREAD

**Discussion by lhilden** [ente-io/ente#2020] [Enhancements]
Before importing codes from other TOTP applications (e.g. Authy), I want the option to either 
1. remove all of the current codes 
2. replace existing codes with new codes

**Comment by freeidiot40x2** [comment]
> Before importing codes from other TOTP applications (e.g. Authy), I want the option to either
> 
>     1. remove all of the current codes
> 
>     2. replace existing codes with new codes

Second this, an option to batch delete the codes would be very helpful.
